python/pyarrow/tests/test_util.py

```diff
@@ -207,3 +209,19 @@ def test_signal_refcycle():
         assert wr() is not None
         _break_traceback_cycle_from_frame(sys._getframe(0))
         assert wr() is None
+
+
+@pytest.mark.skipif(sys.platform != "win32",
+                    reason="Timezone database is already provided.")
+def test_download_tzdata_on_windows():
+    # Download timezone database
+    download_tzdata_on_windows()
+
+    # Inspect the folder
+    tzdata_path = os.path.expandvars(r"%USERPROFILE%\Downloads\tzdata")
+    tzdata_zones_path = os.path.join(tzdata_path, "windowsZones.xml")
+    assert os.path.exists(tzdata_path)
```

Comment:
Can you check that this path does _not_ yet exist before calling `download_tzdata_on_windows` ? 

Because I am wondering that on our windows builds, didn't we already download the data in the setup scripts to ensure we can run the tz tests? In that case this test might not actually be testing that it was downloaded

Reply:
I have added a check and code to remove the data before downloading it again :)
https://github.com/apache/arrow/pull/38179/commits/82f3ecf1e2d7f1597422733396efdc81d2383b91